requirements.txt

```diff
@@ -1,14 +1,66 @@
-pytest
-pytest-asyncio
-pytest-cov
-pytest-mock
-coverage
-requests
-pyjwt[crypto]
-motor
-unidecode
-ldap3
-aio_pika
-flake8
-cachetools
-git+https://github.com/WIPACrepo/rest-tools@v1.1.20#egg=rest_tools
```

Comment:
Some of these may be useful to put into a `requirements-dev.txt` at the root dir.

Reply:
Oh, I see they're in `[options.extras_require].tests`. I'm not sure what's best

Reply:
I guess the question is how much we care about `requirements.txt` files, except as a record of what happened in CI.  If we say the authoritative package list is in `setup.cfg` and change the env setup to just doing `pip install -e .[tests]` then the new way works.

Reply:
> I guess the question is how much we care about `requirements.txt` files, except as a record of what happened in CI. If we say the authoritative package list is in `setup.cfg` and change the env setup to just doing `pip install -e .[tests]` then the new way works.

I agree, I think this is the best way to do things (this is also why I've preferred using tox for tests). The only issue is with the mypy action, it installs from `requirements-dev.txt` to lint things like tests.

Reply:
I added an issue on the mypy action.  Of course, `krs` doesn't actually do a mypy test right now.   You can tell that I wrote this, as it's not strongly typed. :smile: